src/io/iohk/scalanet/peergroup/Subscribers.scala

```diff
@@ -0,0 +1,25 @@
+package io.iohk.scalanet.peergroup
+
+import java.nio.ByteBuffer
+import java.util.concurrent.CopyOnWriteArraySet
+
+import monix.reactive.{Observable, OverflowStrategy}
+import monix.reactive.observers.Subscriber
+
+import scala.collection.mutable
+import scala.collection.JavaConverters._
+
+private[scalanet] class Subscribers {
+  val subscriberSet: mutable.Set[Subscriber.Sync[ByteBuffer]] =
```

Comment:
Can we make this private and create a method to update the set, this would avoid leaking implementation details.